app/aggregates/reviewing.rb

```diff
@@ -5,20 +5,17 @@ def message_key
     end
   end
 
-  class AlreadyCompleted < Error; end
+  # class AlreadyCompleted < Error; end
   class AlreadyMarkedAsReady < Error; end
   class AlreadyReceived < Error; end
   class AlreadySentBack < Error; end
+  class AlreadyReviewed < Error; end
   class DecisionAlreadyLinked < Error; end
-  class CannotCompleteWhenSentBack < Error; end
-  class CannotMarkAsReadyWhenCompleted < Error; end
-  class CannotMarkAsReadyWhenSentBack < Error; end
-  class CannotSendBackWhenCompleted < Error; end
+  class DecisionNotLinked < Error; end
   class IncompleteDecisions < Error; end
-  class NotReceived < Error; end
 
   class DecisionAdded < Event; end
-  class MaatDecisionAdded < Event; end
```

Comment:
We no longer differentiate which type of decision added in the review aggregate.